packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2149,6 +2151,7 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// }
   /// ```
   @protected
+  @mustCallSuper
```

Comment:
not sure about this one. Why should I not be able to create a subclass that throws away all semantic information of the super class?

Reply:
Removed.